src/main/java/com/catapi/view/CatBreedView.java

```diff
@@ -0,0 +1,28 @@
+package com.catapi.view;
+
+import lombok.Getter;
+
+import java.util.Objects;
+
+@Getter
+public record CatBreedView(
+        String id,
+        String name,
+        String origin,
+        String description,
+        String wikipediaUrl
```

Comment:
"wikipediaUrl" filed name doesn't match "wikipedia_url" field in json. So we need to use @JsonProperty annotation